File: Tl2.2 SOAP_DOTNET_EUREKA_BANK_GR08/01.SERVIDOR/ws_eureka_bank_soap_dotnet/ec.edu.monster.service/MovimientoService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ws_eureka_bank_soap_dotnet.ec.edu.monster.DAO;
using ws_eureka_bank_soap_dotnet.ec.edu.monster.model;

namespace ws_eureka_bank_soap_dotnet.ec.edu.monster.service
{
    public class MovimientoService
    {
        private readonly MovimientoDAO _dao = new MovimientoDAO();

        public List<MovimientoModel> ObtenerMovimiento(string cuenta)
        {
            return _dao.ObtenerMovimientosPorCuenta(cuenta);
        }
    }
}

File: Tl2.3 RESTFULL_JAVA_EUREKA_BANK_GR08/01.SERVIDOR/ws_eureka_bank_restful_java/src/main/java/ec/edu/monster/servicio/MovimientoService.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.monster.servicio;

import ec.edu.monster.dao.MovimientoDAO;
import ec.edu.monster.modelo.MovimientoModel;
import java.sql.SQLException;
import java.util.List;

/**
 *
 * @author crist
 */
public class MovimientoService {
    MovimientoDAO DAO = new MovimientoDAO();

    public List<MovimientoModel> ObtenerMovimiento(String cuenta) throws SQLException {
        return DAO.obtenerMovimientosPorCuenta(cuenta);
    }
}
